web/services/createProjectsService.php

```diff
@@ -208,11 +209,37 @@
 
 
     if (count($createProjects) >= 1)
-        if (ProjectsFacade::CreateProjects($createProjects) == -1)
-            $string = "<return service='createProjects'><error id='1'>There was some error while creating the projects</error></return>";
+        if(count($createProjects) == 1){
+            $operationResult = ProjectsFacade::CreateProject($createProjects[0]);
+            if (!$operationResult->isSuccessful)
+                $string = "<return service='createProjects'><error id='1'>". $operationResult->message . "</error></return>";
```

Comment:
We still return code 200 in case of error, we should probably change it with `http_response_code()`. Not your fault, this service didn't do it, but newer code by Ana does :slightly_smiling_face:

Reply:
Here's an example https://github.com/Igalia/phpreport/blob/main/web/services/getHolidays.php#L43